### PR TITLE
Client cert

### PR DESCRIPTION
**Pull request by tobias-**
* Add health checks for pods
* Allow configuring JAVA_OPTS e.g. `-agentlib:jdwp=....`
* Add unary versions of some commands (may help for certain proxies that don't handle grpc fully)
* Change some environment variable names for test-client

### CODE REVIEW

File: kubernetes/deployment-webserver.yaml [hunk 1]
```diff
@@ -27,6 +27,38 @@ spec:
...
+            failureThreshold: 50
+            initialDelaySeconds: 10
+            httpGet:
+              path: /health
+              port: 8070
+              scheme: HTTP
+              httpHeaders:
+                # Make sure health checks aren't traced
+                - name: traceparent
+                  value: 00-11111111111111111111111111111111-2222222222222222-00
```
Comment (pgilmore) [inline kubernetes/deployment-webserver.yaml:61]: Nice!

File: tachikoma-postfix-utils/src/main/kotlin/com/sourceforgery/tachikoma/ClientProvider.kt [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                        sslCtxBuilder.trustManager(InsecureTrustManagerFactory.INSTANCE)
+                    }
+                }
+                .build()
+        )
+        .addHeader("x-apitoken", apiToken)
+        .responseTimeout(Duration.ofDays(365))
+        .writeTimeout(Duration.ofDays(365))
+        .maxResponseLength(0)
+}
```
Comment (pgilmore) [inline tachikoma-postfix-utils/src/main/kotlin/com/sourceforgery/tachikoma/ClientProvider.kt:32]: Format!

**Review by pgilmore — APPROVED** [review]
Approved with comments

File: tachikoma-postfix-utils/src/main/kotlin/com/sourceforgery/tachikoma/ClientProvider.kt (hunk 2 above)
Comment (tobias-) [reply tachikoma-postfix-utils/src/main/kotlin/com/sourceforgery/tachikoma/ClientProvider.kt]: ktlintformat changes nothing. I've added a .editorconfig that will help in intellij et al, but it doesn't change ktlintformat.

**Review by christianekstrand — APPROVED** [review]

**Pull request merged**